### ISSUE TITLE
Unable to send email with any other charset than us-ascii

### ISSUE THREAD

**Issue by mortenlj** [scrapy/scrapy#348]
Sending an email-body with characters that doesn't exist in the us-ascii encoding fails with the following error:

	  File "/home/fimojoha/.virtualenvs/obos-alert/lib/python2.7/site-packages/scrapy/mail.py", line 69, in send
	    dfd = self._sendmail(rcpts, msg.as_string())
	  File "/usr/lib64/python2.7/email/message.py", line 137, in as_string
	    g.flatten(self, unixfrom=unixfrom)
	  File "/usr/lib64/python2.7/email/generator.py", line 83, in flatten
	    self._write(msg)
	  File "/usr/lib64/python2.7/email/generator.py", line 108, in _write
	    self._dispatch(msg)
	  File "/usr/lib64/python2.7/email/generator.py", line 134, in _dispatch
	    meth(msg)
	  File "/usr/lib64/python2.7/email/generator.py", line 180, in _handle_text
	    self._fp.write(payload)
	exceptions.UnicodeEncodeError: 'ascii' codec can't encode character u'\xe6' in position 66: ordinal not in range(128)

Example code that should illustrate the problem (untested because I don't know how to set up the entire twisted environment):

```python
from scrapy.mail import MailSender

mailer = MailSender()
body = u"\u00e5 \u03b1-\u03c9 \u1128 \u114e"
subject = u"Subject"
mailer.send(["example@example.com"], subject, body)
```

**Comment by makalaaneesh** [comment]
Hey, I'm new and want to contribute to scrapy. Was wondering if this issue is still open? The previous PR is not yet merged. Can i work on this ?

**Comment by redapple** [comment]
Fixed via #1662
